source/includes/_contacts.md

```diff
@@ -247,7 +247,7 @@ print(response.text)
 }
 ```
 
-`PUT https://api.apollo.io/v1/contacts`
+`PUT https://api.apollo.io/v1/contacts/YOUR_CONTACT_ID`
```

Comment:
Do, We need to remove 'id' from parameters table below this?

Reply:
Done, please check. @PavanSankavar . Added a descriptor as well.